src/widgets/settingspages/ExternalToolsPage.cpp

```diff
@@ -60,6 +60,12 @@ ExternalToolsPage::ExternalToolsPage()
                                  getSettings()->streamlinkUseCustomPath);
         groupLayout->setWidget(2, QFormLayout::SpanningRole, customPathCb);
 
+        auto note =
+            new QLabel("The custom path may not need the file extension.");
+        note->setWordWrap(true);
+        note->setStyleSheet("color: #bbb");
+        groupLayout->setWidget(3, QFormLayout::SpanningRole, note);
```

Comment:
This is not accurate on Linux. To create a new process on Linux you *have* to have the full file name, extensions are not special.

Reply:
I misunderstood the issue, the problem is not with the custom path but with the name of the executable. I was able to recreate the issue #5073 on linux, where renaming the executable name from `streamlink.AppImage` to `streamlink` fixed the problem. 
So the note message should probably be `If Chatterino is not able to find Streamlink executable, renaming the executable to not have a file extension might help.`, or something like that. Also this message probably should only appear on Linux.

Reply:
It's probably best to say what Chatterino looks for instead, something like "Chatterino expects the binary to be called X" where X should be the name of the streamlink binary depending on the platform